Data/Sys/Load/GraphicMods/Super Smash Bros Brawl/metadata.json

```diff
@@ -0,0 +1,73 @@
+{
+	"meta":
+	{
+		"title": "Framebuffer Fixes",
+		"author": "iwubcode",
+		"description": "Automatically disables 'Store EFB Copies to Texture Only' when a character is offscreen.  Automatically disables 'Store XFB Copies to Texture Only' during the classic mode clear screen."
```

Comment:
Since these descriptions are shown in the Graphics Mods pane I think they should focus on what changes a user will see rather than what they do on a technical level. For example, here I would say `Shows the background behind characters in the image displayed when they are off-screen. Shows the victory portrait instead of a magenta box during the Classic mode clear screen.` or however you want to phrase that.

Reply:
Thanks for the feedback!  I don't feel strongly about this.  I know others feel strongly about how things are worded.  @JosJuice / @JMC47 how do you feel about a more casual explanation instead of a technical one?

Reply:
I would like to keep the mentions of the setting names in there. After all, these settings are user-facing things, so telling the user that they no longer need to go into the settings and manually disable those hacks is useful information to the user. But it could be good to also mention what visible effect this has in the game. I think the current description is kind of esoteric unless you happen to have read the Dolphin wiki page for this game.